tiled/server/authentication.py

```diff
@@ -93,33 +92,14 @@ class TokenData(BaseModel):
     username: Optional[str] = None
 
 
-class APIKeyAuthorizationHeader(APIKeyBase):
-    """
-    Expect a header like
-
-    Authorization: Apikey SECRET
-
-    where Apikey is case-insensitive.
-    """
-
-    def __init__(
-        self,
-        *,
-        name: str,
-        scheme_name: Optional[str] = None,
-        description: Optional[str] = None,
-    ):
-        self.model: APIKey = APIKey(
-            **{"in": APIKeyIn.header}, name=name, description=description
-        )
-        self.scheme_name = scheme_name or self.__class__.__name__
-
+# TODO: remove custom subclass https://github.com/bluesky/tiled/issues/921
+class StrictAPIKeyHeader(APIKeyHeader):
```

Comment:
A docstring to describe the purpose of this subclass (What makes it `Strict`?) would be helpful here.